packages/core/bootstrap/src/lib/server.ts

```diff
@@ -124,3 +127,33 @@ function setupMetricsServer() {
 
   metricsApp.listen(metricsPort, () => logger.info(`Monitoring listening on port ${metricsPort}!`))
 }
+
+const windowMs = 1000 * 5
+const max = parseInt(process.env.SERVER_RATE_LIMIT_MAX || '250') // default to 250 req / 5 seconds max
+const delayAfter = max * (Number(process.env.SERVER_SLOW_DOWN_AFTER_FACTOR) || 0.8) // we start slowing down requests when we reach 80% of our max limit for the current interval
```

Comment:
Any reason we `parseInt()` the others and `Number()` here? Can we be consistent with them?

Reply:
Ah, is it because of decimals?

Reply:
Yes decimals ⬆️